local-dev/render_environment_vars.sh

```diff
@@ -27,6 +27,7 @@ PARTICIPANT_API_ENV_VARS=(
   "REDIRECT_ALL_EMAILS_TO:static:$DEV_EMAIL"
   "B2C_CONFIG_FILE:static:b2c-config.yml"
   "SENDGRID_API_KEY:vault:vault read -field=api_key secret/dsp/ddp/d2p/dev/sendgrid"
+  "SWAGGER_ENABLED:static:true"
```

Comment:
How does this fit in here?

Reply:
We disable the Swagger UI for the Participant API by default because we don't want participants to have access to it. There's no reason to disable it for local development though.

It's not directly related to any of the other changes in this PR, but during development/testing of this PR I ran into a brief headache because I hadn't realized that we disabled it by default. I added it to the setup script so other devs in the future wouldn't run into the same confusion